src/editor/codeeditor.cpp

```diff
@@ -105,6 +109,8 @@ CodeEditor::~CodeEditor()
 
 void CodeEditor::setTextColors()
 {
+    fmtUndefined.setUnderlineStyle(QTextCharFormat::WaveUnderline);
+    fmtUndefined.setUnderlineColor(Qt::red);
```

Comment:
create config parametr with this color. Do the same as for "ftm" variables belowe. You can set this value in the configparamers.h file